docs/getting-started/installation/frc-plugin.md

```diff
@@ -0,0 +1,6 @@
+# Installing the FRC plugin for IntelliJ
+To get things like the right build tasks, class templates, and project templates in IntelliJ you need the FRC plugin. To install this you need to open IntelliJ, and navigate to `File > Settings > Plugins` and then click on the "Marketplace" tab. You're going to want to search "FRC" and download the plugin by Mark Vedder (also shows as Javaru for some people).
+
+Once you've installed it you'll now be able to create a robot project yourself buy going to `File > New > Project` and selecting "FRC Robot Project". You'll be prompted to select things like your chosen template, language, and WPILib version, but all of that is pretty self explanatory if you're familiar with the WPILib VS Code plugin.
```

Comment:
>but all of that is pretty self explanatory if you're familiar

We shouldn't assume anything is obvious, self explanatory, or already known. It'll help us write more complete documentation that is easy to follow for beginners.

Add a link to the VSCode plugin getting started page. https://docs.wpilib.org/en/stable/docs/software/vscode-overview/creating-robot-program.html